### PR TITLE
MWPW-143894 moving category authoring to separate row

### PR DESCRIPTION
**Pull request by npeltier**
resolution proposal to limit authoring errors in sidenav update vs categories

Resolves: [MWPW-143894](https://jira.corp.adobe.com/browse/MWPW-143894)

**Test URLs:**
- Before: https://main--cc--adobecom.hlx.live/drafts/npeltier/sidenav-test-old?martech=off
- After: https://stage--cc--npeltier.hlx.live/drafts/npeltier/sidenav-test

### CODE REVIEW

File: creativecloud/blocks/sidenav/sidenav.js [hunk 1]
```diff
@@ -128,24 +128,31 @@ function appendResources(rootNav, resourceLink) {
 
 export default async function init(el) {
   const libs = getLibs();
+  const rows = Array.from(el.children);
+  if (!rows || rows.length === 0) {
+    return;
```
Comment (yesil) [inline creativecloud/blocks/sidenav/sidenav.js:133]: suggestion:

```javascript
const [mainRow, categoryRow] = el.children;
if (!mainRow) return;
```

**Review by yesil — APPROVED** [review]

**Comment by aishwaryamathuria** [comment]
@npeltier Could you check the failing git checks?

File: creativecloud/blocks/sidenav/sidenav.js (hunk 1 above)
Comment (yesil) [reply creativecloud/blocks/sidenav/sidenav.js:133]: no, it is safe.

**Comment by npeltier** [comment]
@aishwaryamathuria only psi check is now failing, can you please review?

**Comment by npeltier** [comment]
closing that one as stage has been forced pushed with bad code

**Pull request closed without merging**